File: C#/1007 Diferença.cs
using System;

class MainClass {
  public static void Main (string[] args) 
  {
  int A = Convert.ToInt32(Console.ReadLine());
  int B = Convert.ToInt32(Console.ReadLine());
  int C = Convert.ToInt32(Console.ReadLine());
  int D = Convert.ToInt32(Console.ReadLine());
  int DIFERENCA; 

  DIFERENCA = (A * B - C * D);
  Console.WriteLine("DIFERENCA = {0}", DIFERENCA);
    
  }
}

File: C/1007 Diferença.c
#include <stdio.h>

int main(void) 
{
  int a, b, c, d, x;
  scanf("%d %d", &a, &b);
  scanf("%d %d", &c, &d);
  x = a * b - c * d;
  printf("DIFERENCA = %d\n", x);
  return 0;
}